.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Keycloak CI
+
+on: [push]
```

Comment:
@stianst I believe we need the pull request event here. Something like:

```
on: [push, pull_request]
```

Reply:
Updated